tests/serializers/tests.py

```diff
@@ -353,6 +353,34 @@ def test_serialize_proxy_model(self):
         self.assertEqual(base_data, proxy_proxy_data.replace('proxy', ''))
 
 
+class SerializerAPITests(TestCase):
+
+    def test_stream_class(self):
+        class File(object):
+            def __init__(self):
+                self.lines = []
+
+            def write(self, line):
+                self.lines.append(line)
+
+            def getvalue(self):
+                return ''.join(self.lines)
+
+        class Serializer(serializers.json.Serializer):
+            stream_class = File
+
+        score = Score(score=3.4)
+        score.save()
+
+        serializer = Serializer()
+        data = serializer.serialize(Score.objects.all())
+        self.assertIs(serializer.stream_class, File)
```

Comment:
You might want to `assertIsInstance(serializer.stream, File)` to make sure the `stream_class` attribute was actually used.